tools/diagnostics/service_bundle.py

```diff
@@ -0,0 +1,148 @@
+import json
+import logging
+import os
+
+from sdk_utils import groupby
+import sdk_cmd
+import sdk_diag
+
+from bundle import Bundle
+import agent
+import config
+
+log = logging.getLogger(__name__)
+
+
+class ServiceBundle(Bundle):
+    DOWNLOAD_FILES_WITH_PATTERNS = ["^stdout(\.\d+)?(\.log)?$", "^stderr(\.\d+)?(\.log)?$"]
```

Comment:
I don't believe the `.log` extensions are required.

Reply:
01f49021bf51b8c54d7779985e2dc1f776abb341